src/components/List.js

```diff
@@ -3,6 +3,8 @@ import { useHistory } from 'react-router-dom';
 import { FirestoreCollection, withFirestore } from 'react-firestore';
 import Item from './Item';
 import '../styles/List.css';
+import calculateEstimate from '../lib/estimates';
+import secondsToDays from '../lib/secondsToDays';
```

Comment:
I love how the `estimates` and `secondsToDays` are abstracted into the lib folder.